src/index.js

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] As requested by the previous reviewer, the calculator component must be placed in `App.js`, the best you could do is create a new react app and transfer the code from the one you have to the one you will create.

![image](https://user-images.githubusercontent.com/81932478/198068580-67b32e97-435f-4be0-a562-140bf74a2fc6.png)